lib/ufe/wrapUtils.cpp

```diff
@@ -18,13 +18,42 @@
 
 #include "UsdSceneItem.h"
 #include "Utils.h"
+#include "Global.h"
 
 #include <ufe/runTimeMgr.h>
 #include <ufe/rtid.h>
 
 using namespace MayaUsd;
 using namespace boost::python;
 
+namespace {
+
+std::size_t find(
+    const std::string& s, std::size_t size, std::size_t begin, char sep)
+{
+    std::size_t foundPos = s.find(sep, begin);
+    return foundPos == std::string::npos ? size : foundPos;
+}
+
+std::vector<std::string> split(const std::string& src, char sep)
+{
+    std::vector<std::string> names;
+    if (sep) {
+        std::size_t end = 0;
+        const std::size_t size = src.size();
+        for (std::size_t begin = 0; begin < size; begin = end+1) {
+            end = find(src, size, begin, sep);
+            names.push_back(src.substr(begin, end-begin));
+        }
+    }
+    else {
+        names.push_back(src);
+    }
+    return names;
+}
+
+}
+
```

Comment:
Could we not just use `TfStringTokenize()` below instead?:
https://github.com/PixarAnimationStudios/USD/blob/be1a80f8cb91133ac75e1fc2a2e1832cd10d91c8/pxr/base/tf/stringUtils.h#L424

Reply:
Great suggestion, done in
https://github.com/Autodesk/maya-usd/pull/237/commits/f53f29c7258bc1d99897d843dbfef2e0316dd702